torch/csrc/inductor/aoti_runner/model_container_runner.h

```diff
@@ -82,5 +82,27 @@ class TORCH_API AOTIModelContainerRunner {
   std::unique_ptr<torch::aot_inductor::ProxyExecutor> proxy_executor_;
 };
 
+using CreateAOTIModelRunnerFunc = std::shared_ptr<AOTIModelContainerRunner> (*)(
+    const std::string& model_so_path,
+    size_t num_models,
+    const std::string& device_str,
+    const std::string& bin_dir);
```

Comment:
The signature of this function pointer is aligned with the constructor of CUDA `AOTIModelContainerRunnerCUDA`.